src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,20 @@
+package wolox.training.exceptions;
+
+public class BookIdMismatchException extends RuntimeException {
+
+    public BookIdMismatchException() {
+        super("The book's id does not correspond to the data to update");
+    }
+
+    public BookIdMismatchException(final String message, final Throwable cause) {
```

Comment:
I don't think this constructor is necessary, leaving the constructor that accepts a custom message as a parameter is enough.